google-workspace.sgnl.yaml

```diff
@@ -0,0 +1,277 @@
+        type: String
+      - name: givenName
+        externalId: $.name.givenName
+        type: String
+      - name: familyName
+        externalId: $.name.familyName
+        type: String
+      - name: fullName
+        externalId: $.name.fullName
+        type: String
+      - name: isAdmin
+        externalId: isAdmin
+        type: Bool
+      - name: isDelegatedAdmin
+        externalId: isDelegatedAdmin
+        type: Bool
+      - name: lastLoginTime
+        externalId: lastLoginTime
+        type: DateTime
+      - name: creationTime
+        externalId: creationTime
+        type: DateTime
+      - name: agreedToTerms
+        externalId: agreedToTerms
+        type: Bool
+      - name: suspended
+        externalId: suspended
+        type: Bool
+      - name: archived
+        externalId: archived
+        type: Bool
+      - name: changePasswordAtNextLogin
+        externalId: changePasswordAtNextLogin
+        type: Bool
+      - name: ipWhitelisted
+        externalId: ipWhitelisted
+        type: Bool
+      - name: nonEditableAliases
+        externalId: nonEditableAliases
+        type: String
+        list: true
+      - name: customerId
+        externalId: customerId
+        type: String
+      - name: orgUnitPath
+        externalId: orgUnitPath
+        type: String
+      - name: isMailboxSetup
+        externalId: isMailboxSetup
+        type: Bool
+      - name: isEnrolledIn2Sv
+        externalId: isEnrolledIn2Sv
+        type: Bool
+      - name: isEnforcedIn2Sv
+        externalId: isEnforcedIn2Sv
+        type: Bool
+      - name: includeInGlobalAddressList
+        externalId: includeInGlobalAddressList
+        type: Bool
+      - name: recoveryEmail
+        externalId: recoveryEmail
+        type: String
+  UserEmail:
+    parent: User
+    displayName: UserEmail
+    externalId: emails
+    description: Entity representing an email object for a Google Workspace User
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: address
+        externalId: address
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+  # https://developers.google.com/admin-sdk/directory/reference/rest/v1/groups#Group
+  Group:
+    displayName: Group
+    externalId: Group
+    description: Entity representing a Google Workspace Group
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: email
+        externalId: email
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        attributeAlias: groupId
+      - name: kind
+        externalId: kind
+        type: String
+      - name: etag
+        externalId: etag
+        type: String
+      - name: name
+        externalId: name
+        type: String
```

Comment:
We might want to index this if one wants to filter by group name in snippets

Reply:
added